core/test/usBundleRegistryPerformanceTest.cpp

```diff
@@ -64,7 +64,9 @@ namespace
         InstallTestBundleNoErrorHandling(fmc, "TestBundleM");
         InstallTestBundleNoErrorHandling(fmc, "TestBundleR");
         InstallTestBundleNoErrorHandling(fmc, "TestBundleRA");
+#ifndef US_DISABLE_TESTING_LINKED_RESOURCES // FIXME: TestBundleRL uses linker to embed manifest.json on Windows
```

Comment:
please create an issue so we can circle back to address linked resources and how it affects installing bundles.